src/components/MapPanel.vue

```diff
@@ -368,6 +368,13 @@
           return null
         }
       },
+      basemapSelectControlPosition() {
+        if (this.isMobileOrTablet) {
+          return 'topright'
+        } else {
+          return 'topalmostright'
```

Comment:
Note that we don't need an else if we have a `return` in the other branch of the conditional.